### PR TITLE
Add CLI to interface with new internal Go API

### PR DESCRIPTION
**Pull request by AndreasPB**
# Description

* Sets up CLI to use the new internal API that's in the making

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by AndreasPB** [comment]
Performance hyyyype
151k media fetched and stored in 3,5min
image

File: backend/app/api.py [hunk 1]
```diff
@@ -79,6 +83,52 @@ def fetch_jsongz_files():
...
+            f"{tmdb_url}movie/changes?api_key={tmdb_key}&"
+            f"end_date={today}&start_date={yesterday}&page=1"
+        )
+        tv_res = client.get(
+            f"{tmdb_url}tv/changes?api_key={tmdb_key}&"
+            f"end_date={today}&start_date={yesterday}&page=1"
+        )
+
+    movie_ids = [f"m{movie['id']}" for movie in movie_res.json()["results"]]
+    tv_ids = [f"t{tv['id']}" for tv in tv_res.json()["results"]]
```
Comment (nullxDEADBEEF) [inline backend/app/api.py:127]: Check for adult :D

Comment (AndreasPB) [reply backend/app/api.py]: DONE

**Comment by AndreasPB** [comment]
Ask TMDB if a change in providers triggers the "changed"-whatever

**Review by Primdahl26 — APPROVED** [review]
Awesome, just a few things

**Comment by AndreasPB** [comment]
UX boii

image

File: backend/cli.py [hunk 2]
```diff
@@ -84,6 +88,37 @@ def index_meilisearch():
     update_index()
 
 
+@app.command()
+def update_ids(ids: list[str]):
+    with httpx.Client() as client:
+        client.post("http://internal:8888/update-media", json={"ids": ids})
```
[the remark this answers is not in the document]
Comment (AndreasPB) [reply backend/cli.py]: Done, check image below

File: backend/cli.py [hunk 3]
```diff
@@ -84,6 +88,37 @@ def index_meilisearch():
...
+
+
+@app.command()
+def update_media(
+    chunk_size: int = 1000, first_time: bool = False, popularity: float = 0
+):
+    """Sends media ids to our internal update-media endpoint in chunks"""
```
[the remark this answers is not in the document]
Comment (AndreasPB) [reply backend/cli.py:110]: Done, check image below

**Pull request merged**